### ISSUE TITLE
Core Dumped

### ISSUE THREAD

**Issue by leonardoaraujosantos** [Microsoft/CNTK#501]
Hi Guys I'm training with a large zip file as training 200Gb (with images with different sizes inside the zip), and I'm having this error...
## leo@monsterpc2:~/cntk/Examples/Image/Miscellaneous/Apical$ cntk configFile=ResNet_18.cntk

Build info: 

```
    Built time: Apr 25 2016 04:32:40
    Last modified date: Mon Apr 25 04:09:36 2016
    Build type: release
    Build target: GPU
    With 1bit-SGD: no
    Math lib: acml
    CUDA_PATH: /usr/local/cuda-7.5
    CUB_PATH: /usr/local/cub-1.4.1
    CUDNN_PATH: /usr/local/cudnn-4.0
    Build Branch: HEAD
    Build SHA1: 29c065ae7c1c370d8852c283d4e9f1a068ba2233
    Built by philly on f3f64d5eed9a
    Build Path: /home/philly/jenkins/workspace/CNTK-Build-Linux
```

---

Redirecting stderr to file ./Output/ResNet_18_Train.log
Aborted (core dumped)
###### #############LOG

Allocating matrices for forward and/or backward propagation.
No PreCompute nodes found, skipping PreCompute step.

Starting Epoch 1: learning rate per sample = 0.033333  effective momentum = 0.900000  momentum as time constant = 284.7 samples

Starting minibatch loop.
terminate called after throwing an instance of 'std::bad_weak_ptr'
  what():  bad_weak_ptr
###### 

[ResNet_18.ndl.txt](https://github.com/Microsoft/CNTK/files/275012/ResNet_18.ndl.txt)
[ResNet_18.cntk.txt](https://github.com/Microsoft/CNTK/files/275011/ResNet_18.cntk.txt)

**Comment by Alexey-Kamenev** [comment]
Hello,
Have you tried running CNTK with just a small subset of the data? 
What compression method did you use when creating the zip file (it is recommended to use STORE)?

**Comment by leonardoaraujosantos** [comment]
Hi, thanks I've recreated the Train file without the zip, but still crashes with the following log

############################################################
14 out of 192 nodes do not share the minibatch layout with the input data.

Post-processing network complete.

Loaded model with 192 nodes on GPU 2.

Training criterion node(s):
	CE = CrossEntropyWithSoftmax

Evaluation criterion node(s):

	Err = ErrorPrediction


Allocating matrices for forward and/or backward propagation.
No PreCompute nodes found, skipping PreCompute step.

Starting Epoch 1: learning rate per sample = 0.033333  effective momentum = 0.900000  momentum as time constant = 284.7 samples

Starting minibatch loop.
terminate called without an active exception
terminate called recursively

**Comment by Alexey-Kamenev** [comment]
Hmm, I see that you already reduced minibatch size. Just for the sake of experiment, can you try even smaller minibatches? E.g. 16 or 8? Though when CNTK runs out of GPU memory the error looks different, it's worth trying anyway.
Also, as I understand, you are using your own dataset, right? Have you tried using ILSVRC-2012 (ImageNet)? 
Also, what kind of GPUs are you using?
And the last one: have you tried using the latest version from the master branch?

**Comment by leonardoaraujosantos** [comment]
Hi Alexey it's a tesla K-80, I've downloaded the release version, I will try to compile from the master branch. Thanks

**Comment by mahilleb-msft** [comment]
@leonardoaraujosantos - assuming this is fixed for you now? Please re-open or file a new issue if necessary. Testing with latest bits / ResNet models is also possible, there have been many changes since last year.
